PlaceRouteHierFlow/placer/Placer.cpp

```diff
@@ -527,10 +527,20 @@ std::map<double, std::pair<SeqPair, ILP_solver>> Placer::PlacementCoreAspectRati
   double curr_cost = 0;
   int trial_count = 0;
   int max_trial_count = 10000;
+
+  unsigned int seed = 0;
+  if (hyper.SEED > 0) {
+    seed = hyper.SEED;
+    srand(0);
```

Comment:
Don't we want `srand(seed)`?

Reply:
Yes, absolutely. This is a left-over from trials. Fixed in the last commit.